File: Java/Area.java
import java.util.Scanner;

/**
 * Area
 */
public class Area {

    public static void main(String[] args) {
        Scanner raf = new Scanner(System.in);
        double a = raf.nextDouble();
        double b = raf.nextDouble();
        double c = raf.nextDouble();

        System.out.printf("TRIANGULO: %.3f\n",(a * c)/2);
        System.out.printf("CIRCULO: %.3f\n",(3.14159 * Math.pow(c, 2)));
        System.out.printf("TRAPEZIO: %.3f\n",((a + b) * c)/2);
        System.out.printf("QUADRADO: %.3f\n",(Math.pow(b, 2)));
        System.out.printf("RETANGULO: %.3f\n",a * b);

    }
}

File: Python/Area.py
import math

a,b,c = map(float, input().split(" "))

#A
areaTriangle = (a * c) / 2 
print("TRIANGULO: %.3f"%areaTriangle)

#B
areaCircle = math.pow(c, 2) * 3.14159
print("CIRCULO: %.3f"%areaCircle)

#C
areaTrapesium = ((a + b) * c) / 2
print("TRAPEZIO: %.3f"%areaTrapesium)

#D
areaSquare = math.pow(b, 2)
print("QUADRADO: %.3f"%areaSquare)

#E
areaRectangle = a * b
print("RETANGULO: %.3f"%areaRectangle)
